lib/exceptions.rb

```diff
@@ -6,4 +6,5 @@ class RBACError < StandardError; end
   class KieError < StandardError; end
   class InvalidStateTransitionError < StandardError; end
   class UserError < StandardError; end
+  class NegativeSequenceError < StandardError; end
```

Comment:
Recommend using just `NegativeSequence`

Reply:
It is a personal preference. I tend to append Error in the end so that we can clearly know it is an error class different from regular class, just like in the Java world every exception class ends with `Exception`. I can remove the trailing world.

Reply:
I think that adding `Error` to the end depends on the actual error. For example if it's another process that is having the problem, then it makes sense, like `TopologyError` or in the case of these here, `KieError`, or `RBACError`. But if it's a state that we came up with, I think dropping the `Error` suffix makes sense. `InvalidStateTransition`, `NegativeSequence`. 

This follows what we're doing in catalog with stuff like `InvalidSurvey`, `OrderUncancelable`, but we have `SourcesError` and `TopologyError` and `ApprovalError`.

I don't have a super strong opinion on this, I could go either way, just pointing out how we currently handle these.